skimage/filters/_frangi.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+
+__all__ = ['frangi', 'hessian']
+
+
+def _frangi_hessian_common_filter(image, scale, scale_step, beta1, beta2):
+    """This is an intermediate function for Frangi and Hessian filters.
+
+    Shares the common code for Frangi and Hessian functions.
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Array with input image data.
+    scale : tuple of floats, optional
+        The range of sigmas used.
+    scale_step : float, optional
+        Step size between sigmas.
+    beta1 : float, optional
+        Frangi correction constant.
+    beta2 : float, optional
+        Frangi correction constant.
+    black_ridges : boolean, optional
+        If True (default), detects black ridges, if False - white ones.
```

Comment:
Ach, sorry, one more: remove this kwarg from the doc.

And if it's not too hard, can you squash with git rebase -i?

I am also happy to take over at this point if you are tired. ;)